app/backend/wells/stack.py

```diff
@@ -166,7 +166,8 @@ def _stack(self, records, well: Well) -> Well:
             serializer = submissions.serializers.WellSubmissionStackerSerializer(submission)
             for source_key, value in serializer.data.items():
                 # We only consider items with values, and keys that are in our target
-                if value:
+                # an exception is STAFF_EDIT submissions (we need to be able to accept empty values)
+                if value or value == "" or value is False:
```

Comment:
Does value == "" not mess up other submissions? Like, if an alteration follows a construction, with a blank string, would the alteration not override what was in the construction?